server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -116,7 +114,8 @@ export function configureWebSocketServices(
     clientManager: core.IClientManager,
     metricLogger: core.IMetricClient,
     logger: core.ILogger,
-    maxNumberOfClientsPerDocument: number = 1000000) {
+    maxNumberOfClientsPerDocument: number = 1000000,
+    isTokenExpiryEnabled: boolean = false) {
```

Comment:
Where are you fetching this value from the config?

Reply:
Ditto. You need to wire that in from where the lambdas are invoked.

Reply:
Had this change locally but not checked in. Thanks for the catch! The consumption is in [routerlicious-base/src/alfred/runner.ts](https://github.com/znewton/FluidFramework/blob/r11s-token-expiry/server/routerlicious/packages/routerlicious-base/src/alfred/runner.ts#L62-L75).